src/Stratis.Bitcoin.Features.Consensus/CoinViews/InMemoryCoinView.cs

```diff
@@ -48,23 +54,23 @@ public override Task<FetchCoinsResponse> FetchCoinsAsync(uint256[] txIds, Cancel
                     if (result[i] != null)
                         result[i] = result[i].Clone();
                 }
-                return Task.FromResult(new FetchCoinsResponse(result, this.blockHash));
+                return Task.FromResult(new FetchCoinsResponse(result, this.tipHash));
```

Comment:
Could have introduce empty line above this

Reply:
Fixed